akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -380,12 +400,12 @@ private[http] object HttpServerBluePrint {
   }
 
   private case class ProtocolSwitchShape(
```

Comment:
ah, this will need to be adapted heavily: once #19243 is merged the websocket setup is gone and this stage is a lot cleaner